x-pack/platform/plugins/shared/cases/public/components/add_comment/index.test.tsx

```diff
@@ -316,3 +316,104 @@ describe('draft comment ', () => {
     });
   });
 });
+
+describe('submit comment by key press', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  beforeAll(() => {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    jest.clearAllTimers();
+  });
+
+  afterAll(() => {
+    jest.useRealTimers();
+  });
+  it('should add a comment on the correct key stroke - ctrl + enter', async () => {
```

Comment:
Instead of copying the test code twice, you can use [`test.each`](https://jestjs.io/docs/api#testeachtablename-fn-timeout). Check out [this case test](https://github.com/elastic/kibana/blob/96c60d1172fa7007aa1a53ba11e8da7b4917a006/x-pack/platform/plugins/shared/cases/public/components/connectors/swimlane/validator.test.ts#L46) as an example.

Something along the lines of:

```
test.each(['ctrlKey', 'metaKey'])(
  "should add a comment on the correct key stroke - %s + enter",
  (modifierKey) => {
    (...)
    fireEvent.keyDown(screen.getByLabelText('caseComment'), {
      key: 'Enter',
      code: 13,
      [modifierKey]: true,
    });
})